.dockerignore

```diff
@@ -0,0 +1,5 @@
+.git
+.gitignore
+LICENSE
+local.env.dist
+README.md
```

Comment:
Would this be easier?
```suggestion
# Ignore everything
*

# Whitelist required files
!sync-s3-to-b2.sh
```

Reply:
> Would this be easier?

There's a Dockerfile too.

Reply:
No need to include Dockerfile

Reply:
It seems odd to me to use a file named `ignore` to list files that you do **not** want to ignore.  If I were to add another file to the Docker image in the future, I'm not convinced that I'd remember to add it to the `.dockerignore`.  To me, ignoring a few files is a buffing/polishing/cleanup activity that saves a few milliseconds during the build.

Reply:
The same could be said for remembering to ignore a file. If you forget to whitelist a file, you know pretty quickly. But this is all splitting hairs for a small project like this.